### PR TITLE
OAK-10230: switch oak-authorization-principalbased to shaded guava

### PR DESCRIPTION
**Pull request by reschke**

### CODE REVIEW

File: oak-authorization-principalbased/src/main/java/org/apache/jackrabbit/oak/spi/security/authorization/principalbased/impl/AbstractEntry.java [hunk 1]
```diff
@@ -16,7 +16,7 @@
  */
 package org.apache.jackrabbit.oak.spi.security.authorization.principalbased.impl;
 
-import com.google.common.base.Objects;
+import org.apache.jackrabbit.guava.common.base.Objects;
```
Comment (anchela) [inline oak-authorization-principalbased/src/main/java/org/apache/jackrabbit/oak/spi/security/authorization/principalbased/impl/AbstractEntry.java:19]: in other PR this was replaced by MoreObjects, no?

File: oak-authorization-principalbased/src/main/java/org/apache/jackrabbit/oak/spi/security/authorization/principalbased/impl/ImmutablePrincipalPolicy.java [hunk 2]
```diff
@@ -16,7 +16,7 @@
  */
 package org.apache.jackrabbit.oak.spi.security.authorization.principalbased.impl;
 
-import com.google.common.base.Objects;
+import org.apache.jackrabbit.guava.common.base.Objects;
```
Comment (anchela) [inline oak-authorization-principalbased/src/main/java/org/apache/jackrabbit/oak/spi/security/authorization/principalbased/impl/ImmutablePrincipalPolicy.java:19]: same here

**Review by anchela — APPROVED** [review]
lgtm.... i just have 1 question: in other modules Object was replaced by MoreObjects. why not here?

File: oak-authorization-principalbased/src/main/java/org/apache/jackrabbit/oak/spi/security/authorization/principalbased/impl/AbstractEntry.java (hunk 1 above)
Comment (reschke) [reply oak-authorization-principalbased/src/main/java/org/apache/jackrabbit/oak/spi/security/authorization/principalbased/impl/AbstractEntry.java:19]: It's a specific method in "Objects" that moved to "MoreObjects". Others stayed where they were.

**Pull request merged**